src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -98,12 +96,10 @@ private void validateContent(final CaseDataContent content,
             int length = referenceArray.length;
             String nodeReference = length > 1 ? referenceArray[length - 2] : referenceArray[length - 1];
             List<JsonNode> parentNodes = caseFieldNode.findParents(nodeReference);
-            parentNodes.stream().forEach(parentNode -> {
+            parentNodes.forEach(parentNode -> {
                 JsonNode orgPolicyNode = findOrgPolicyNode(nodeReference, parentNode, length);
                 if (orgPolicyNode.isNull()) {
                     errorList.add(caseFieldId + " cannot have an empty value.");
-                } else if (!defaultValue.equalsIgnoreCase(orgPolicyNode.textValue())) {
```

Comment:
We are just validating null value, what if user changes the OrgPolicyRole, that means if user enters any other value in this field, we need to check whether that case-role exists or not

Reply:
Confirmed with Nigel we need it. Implementing.

Reply:
Done